apps/remix-ide/src/app/panels/tab-proxy.js

```diff
@@ -41,6 +41,13 @@ export class TabProxy extends Plugin {
 
       if (this.fileManager.mode === 'browser') {
         name = name.startsWith(workspace + '/') ? name : workspace + '/' + name
+        // If deleted file is not current file and not an active tab in editor,
+        // ensure current file is active in the editor
+        if (this.fileManager.currentFile() && name !== this.fileManager.currentFile()) {
+          const currentFile = this.fileManager.currentFile()
+          const currentFileTabPath = currentFile.startsWith(workspace + '/') ? currentFile : workspace + '/' + currentFile
+          this.removeTab(name, { name: currentFileTabPath })
```

Comment:
you should probably return here, otherwise you'll call `removeTab` 2 times.

Reply:
fixed